doc/api/perf_hooks.md

```diff
@@ -0,0 +1,454 @@
+* Returns: {Array}
+
+Returns a list of all `PerformanceEntry` objects in chronological order
+with respect to `startTime` whose `performanceEntry.entryType` is equal to
+`type`.
+
+### performance.mark([name])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `name` {string}
+
+Creates a new `PerformanceMark` entry in the Performance Timeline. A
+`PerformanceMark` is a subclass of `PerformanceEntry` whose
+`performanceEntry.entryType` is always `'mark'`, and whose
+`performanceEntry.duration` is always `0`. Performance marks are used
+to mark specific significant moments in the Performance Timeline.
+
+### performance.measure(name, startMark, endMark)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `name` {string}
+* `startMark` {string}
+* `endMark` {string}
+
+Creates a new `PerformanceMeasure` entry in the Performance Timeline. A
+`PerformanceMeasure` is a subclass of `PerformanceEntry` whose
+`performanceEntry.entryType` is always `'measure'`, and whose
+`performanceEntry.duration` measures the number of milliseconds elapsed since
+`startMark` and `endMark`.
+
+The `startMark` argument may identify any *existing* `PerformanceMark` in the
+the Performance Timeline, or *may* identify any of the timestamp properties
+provided by the `PerformanceNodeTiming` class. If the named `startMark` does
+not exist, then `startMark` is set to `timeOrigin` by default.
+
+The `endMark` argument must identify any *existing* `PerformanceMark` in the
+the Performance Timeline or any of the timestamp properties provided by the
+`PerformanceNodeTiming` class. If the named `endMark` does not exist, an
+error will be thrown.
+
+### performance.nodeFrame
+<!-- YAML
+added: REPLACEME
+-->
+
+* {PerformanceFrame}
+
+An instance of the `PerformanceFrame` class that provides performance metrics
+for the event loop.
+
+### performance.nodeTiming
+<!-- YAML
+added: REPLACEME
+-->
+
+* {PerformanceNodeTiming}
+
+An instance of the `PerformanceNodeTiming` class that provides performance
+metrics for specific Node.js operational milestones.
+
+### performance.now()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {number}
+
+Returns the current high-resolution millisecond timestamp.
+
+### performance.timeOrigin
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The [`timeOrigin`][] specifies the high-resolution millisecond timestamp from
+which all performance metric durations are measured.
+
+## Class: PerformanceEntry
+<!-- YAML
+added: REPLACEME
+-->
+
+The base class for all objects in the Performance Timeline.
+
+### performanceEntry.duration
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The total number of milliseconds ellapsed for this entry. This value will not
+be meaningful for all Performance Entry types.
+
+### performanceEntry.name
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The name of the performance entry.
+
+### performanceEntry.startTime
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The high resolution millisecond timestamp marking the starting time of the
```

Comment:
In few other places in the documentation, high resolution is hyphenated. Perhaps we should unhyphenate them?